services/src/pro/users/userdb.rs

```diff
@@ -133,4 +137,20 @@ pub trait UserDb: Send + Sync {
         user: &UserId,
         new_available_quota: i64,
     ) -> Result<()>;
+
+    /// Add a new role
+    ///// TODO: move this method to some AdminDb?
+    async fn add_role(&self, role_name: &str) -> Result<RoleId>;
```

Comment:
either that, or create a AdminDb that also contains stuff like increase_quota. I think we can do it in a separate PR though. You can add it to the success critera of the card if you want. Otherwise we can do it also at a alter point in time.

Reply:
i think the traits can be seperate but the implementation can be for the same struct

Reply:
c3312c0c814965aefcafa48d32c841a76772f137